modules/op-installing-pipelines-as-code-cli.adoc

```diff
@@ -9,13 +9,13 @@
 [role="_abstract"]
 Cluster administrators can use the `tkn-pac` and `opc` CLI tools on local machines or as containers for testing. The `tkn-pac` and `opc` CLI tools are installed automatically when you install the `tkn` CLI for {pipelines-title}.
 
-You can install the `tkn-pac` and `opc` version `1.9.0` binaries for the supported platforms:
+You can install the `tkn-pac` and `opc` version `1.9.1` binaries for the supported platforms:
 
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.0/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.0/tkn-linux-s390x.tar.gz[Linux on IBM Z and LinuxONE (s390x)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.0/tkn-linux-ppc64le.tar.gz[Linux on IBM Power (ppc64le)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.0/tkn-macos-amd64.tar.gz[macOS]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.0/tkn-windows-amd64.zip[Windows]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.1/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.1/tkn-linux-s390x.tar.gz[Linux on IBM zSystems and IBM(R) LinuxONE (s390x)]
```

Comment:
```suggestion
* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.1/tkn-linux-s390x.tar.gz[Linux on IBM Z Systems and IBM(R) LinuxONE (s390x)]
```

I hope this is correct. If yes, please treat this as a global comment for all similar instances in this PR. Else, kindly ignore my suggestion.

Reply:
Yes it's correct we have a naming change to IBM zSystems and the IBM(R) that always has to accompany LinuxONE now.   
Once this PR is merged for the 1.9.1 GA I will even add the files to my other PR that does a sweep to add the common attributes. https://github.com/openshift/openshift-docs/pull/56320